recipes/gtf_to_genes/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
+if errorlevel 1 exit 1
```

Comment:
@jj-umn please don't include these bat files. We do not compile for windows.
Let's merge it for now.